src/main/java/com/google/sps/data/Organization.java

```diff
@@ -47,4 +52,89 @@ public Organization(Entity entity) {
      this.urlLink = (String) entity.getProperty("orgUrl");
      this.moderators = (ArrayList) entity.getProperty("moderatorList");
   }
+
```

Comment:
instead of storing the data in separate variables as you do above (and the commit function where you set them directly), what about making these into functions?


example:
constructor() {
  this.entity = entity;
}

long id() {
   return (long) entity.getKey().getId();
}
String name() {
   return entity.getProperty("orgName");
}
etc